portfolio/src/main/webapp/index.html

```diff
@@ -59,7 +59,9 @@ <h2 id="contact">Contact</h2>
       </div>
       <!-- Play around with Java Servlets and try to let the client user enter their name -->
       <h2 id="comments">Leave your comments</h2>
-      <form action="/comments" method="POST">
+      <p id="comments-login">To see and publish comments, please login <a href="/login">here</a></p>
+      <p id="comments-logout">You can log out of your account <a href="/login">here</a></p>
```

Comment:
Here also: it's better to include more context about what the hyperlink does inside the `a` tag.

Reply:
I replace the two "here" links with a "login here" link and a "log out here" link in the newest commit.